### DISCUSSION TITLE
Option to suppress welcome message

### DISCUSSION THREAD

**Discussion by Helvio88** [DblK/tinshop#33] [Ideas]
Just as an idea, an entry in the config file to suppress the shop welcome message.
Maybe `suppressWelcome: true/false`
or an empty message (or nonexisting config key) defaults to supression.

**Comment by DblK** [comment]
Could be good. It's mainly use to ensure everything is loaded in `tinfoil` but I could manage to add an option.

**Comment by DblK** [comment]
This has been done here https://github.com/DblK/tinshop/pull/38
